.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: objective-c
-osx_image: xcode7.3
+osx_image: xcode8.0
```

Comment:
You are perfectly right.
I'll add it and check why travis cannot find the simulator